airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryStagingConsumerFactory.java

```diff
@@ -79,71 +73,52 @@ private long getBigQueryBufferMemoryLimit() {
     return (long) (Runtime.getRuntime().maxMemory() * 0.4);
   }
 
-  private Map<StreamDescriptor, BigQueryWriteConfig> createWriteConfigs(final JsonNode config,
-                                                                        final ConfiguredAirbyteCatalog catalog,
-                                                                        final ParsedCatalog parsedCatalog,
-                                                                        final Function<JsonNode, BigQueryRecordFormatter> recordFormatterCreator,
-                                                                        final Function<String, String> tmpTableNameTransformer) {
+  private Map<StreamDescriptor, StreamConfig> createWriteConfigs(final ConfiguredAirbyteCatalog catalog,
```

Comment:
can this just be `parsedCatalog.getStreams().stream()`? afaict we don't actually need the raw protocol models for anything

(and then we don't need to plumb the raw configured catalog into this method)